charts/quickwit/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: quickwit
 description: Sub-second search & analytics engine on cloud storage.
 type: application
-version: 0.6.5
+version: 0.7.0
 appVersion: "v0.8.1"
```

Comment:
can you update the app version to 0.8.2 too?

Reply:
Yeah sure

Reply:
Done